src/resources/PlatformResources.ts

```diff
@@ -26,6 +28,7 @@ class PlatformResources {
     cluster: Cluster;
     group: Group;
     organization: Organization;
+    indexes: Indexes;
```

Comment:
Every other resource is singular

```suggestion
    index: Indexes;
```